contracts/GovernanceDecisions.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.6.7;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract GovernanceDecisions is AccessControl {
+
+    bytes32 public constant GOVERNOR_ROLE = keccak256("GOVERNOR_ROLE");
+
+    uint8 public marketplaceFee;
+    string[] private marketplaceCategories;
+    string[] private memberships;
+
+    event MarketplaceFeeChanged(uint8 oldFee, uint8 newFee);
+    event MarketplaceCategoryAdded(string category);
+    event MemberAdded(string member);
+
+    constructor(address governor_) public {
+        _setupRole(GOVERNOR_ROLE, governor_);
+        marketplaceFee = 1; // set default marketplace fee
+    }
+
+    function setMarketplaceFee(uint8 fee) public {
```

Comment:
for this and other functions, recommend using external over public as we don't expect to ever call them from within this contract: https://ethereum.stackexchange.com/questions/19380/external-vs-public-best-practices

"The difference is because in public functions, Solidity immediately copies array arguments to memory, while external functions can read directly from calldata. Memory allocation is expensive, whereas reading from calldata is cheap."

with new versions of solidity, for external functions, you also use calldata instead of memory (eg: for the category string) - see https://docs.soliditylang.org/en/v0.5.3/types.html#data-location for more insights